Getting-Started/Backoffice/index.md

```diff
@@ -2,17 +2,18 @@
 These items are common terms and concepts that are used throughout the Umbraco backoffice.
 
 ### [Login screen](Login/)
-When you go to the backoffice for the first time, you're presented with the login screen.
-![Login screen](images/umbraco7-6_login.jpg "The login screen has a greeting, username/password field and optionally a 'Forgotten password' link.")
+When you go to the backoffice for the first time, you're presented with the login screen. [Read more..](Login/)
```

Comment:
I was recently informed by @dampee that having the link in the end as a read more thing was not great for accessibility purposes, would you mind changing this:
login screen. [Read more..](Login/)
To something like this:
[login screen](Login/).

And then also doing it throughout the document? 😉
We are working on a contribution guide and possibly a style guide to cover things like this - sorry it's not available anywhere yet 😬